### DISCUSSION TITLE
Repeated information in release notes from pull request expansion when showing release notes elsewhere in github

### DISCUSSION THREAD

**Discussion by LyleDavis** [community/community#49911] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

When the release notes are generated via the github automated release notes generation, it generates them in something like the following format:

```
### what's changed
* PR title here by @username in #1234
```

This works really well, but if you paste these release notes elsewhere in github the link to the PR will be expanded to show the whole title of the pull request which becomes messy and causes duplication. For example, if you have paste the release notes into a github comment somewhere (or in my case, the description of a PR in another repository for our continuous deployment mechanism) you end up with something like:

```
### what's changed
* PR title here by @username in PR title here repo#number
```

It would be good if there were an option to turn off the hyperlink expansion for particular repositories to avoid the unfortunate duplication.

**Comment by jameshasung** [comment]
It sounds like you are experiencing an issue with the link expansion causing duplication when the release notes are pasted elsewhere on Github.

While I am not able to make any changes to the Github platform, I can suggest a possible workaround for this issue. One option would be to manually edit the release notes after they are generated to remove the link expansion for the specific repositories where duplication is occurring.

**Reply by LyleDavis** [reply]
yeah I know there are manual ways round it, I'm just wondering whether it would be possible to introduce a setting in the repository settings that turns off the link expansion so we don't need manual intervention in the CI/CD flows.

**Reply by jameshasung** [reply]
It is possible to introduce a setting in the repository settings to turn off link expansion, so that manual intervention is not required in the CI/CD flows.

GitHub provides a feature called "Autolink references" which automatically converts certain text strings into links. This feature can be turned off at the repository level by going to the repository's "Settings" tab, then navigating to "Options" and scrolling down to the "Features" section. From there, you can uncheck the box next to "Autolink references" to turn off this feature.

By turning off autolink references, you can ensure that links in pull requests or other documents are not automatically expanded, which can help prevent unintended actions or security risks. However, it is important to note that turning off this feature may also impact the functionality of other tools or integrations that rely on link expansion, so it is recommended to carefully consider the implications before making any changes.
